specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2019-05-01/containerregistry.json

```
@@ -1617,17 +1620,19 @@
       "properties": {
         "days": {
           "format": "int32",
-          "description": "The number of days to retain manifest before it expires.",
+          "description": "The number of days to retain an untagged manifest after which it gets purged.",
+          "default": 7,
           "type": "integer"
         },
         "lastUpdatedTime": {
           "format": "date-time",
-          "description": "The timestamp when the policy was last updated.",
+          "description": "The timestamp when the the policy was last updated.",
```

Adding "the" a second time
```suggestion
          "description": "The timestamp when the policy was last updated.",
```